content/docs/concepts/policies-and-telemetry/istio-metrics.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Istio Metrics
+description: Metrics exported from Istio through Mixer.
+weight: 50
+---
+
+Istio exports metrics through Mixer. They can be found [here](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/config.yaml)
+under the section with “kind: metric”.
+
+We will describe metrics first and then the labels for each metric.
+
+## Metrics
+
+### Request Count
+
+This is a COUNTER metric incremented for a new request to Istio.
```

Comment:
Nit: this counter isn't incremented for a new request to Istio. It is incremented for every request handled by an Istio proxy.

Reply:
Fixed